en/development/configuration.rst

```diff
@@ -71,6 +71,9 @@ instead use the ``config/.env.example`` as a template with placeholder values so
 everyone on your team knows what environment variables are in use and what
 should go in each one.
 
+Before being able to use the ``env()`` syntax, you will need to uncomment 
+the code block on line 59-65 in ``config/bootstrap.php``.
```

Comment:
```suggestion
related code block in ``config/bootstrap.php``.
```

Currently it is lines 61-67, and might change again in the future.

Reply:
Thanks! Was already wondering how to phrase that best.